src/modules/common/components/CleanAnchor.js

```diff
@@ -1,8 +1,33 @@
 // @flow
 
+import * as React from 'react'
 import styled from 'styled-components'
 
-export default styled.a`
+const StyledAnchor = styled.a`
   color: inherit;
   text-decoration: none;
 `
+
+type PropsType = {|
+  newTab?: boolean,
+  href: string,
+  children: React.Node
+|}
+
+class CleanAnchor extends React.PureComponent<PropsType> {
+  static defaultProps = {
+    newTab: true
+  }
+
+  render () {
+    const {newTab, href, children} = this.props
+
+    if (newTab) {
+      return <StyledAnchor href={href} target='_blank' rel='noreferrer'>{children}</StyledAnchor>
```

Comment:
The CleanAnchor should pass all its other props to the StyledAnchor. Otherwise we cannot restyle it with sth like:
```js
styled(CleanAnchor)`
  some: css;
`
```
At least, that's what I believe^^

Reply:
Yeah, if we want to style it, we have to pass the className to children. Currently, we are not styling a CleanAnchor anywhere though. And right now, we only pass className if we really want to style it and not just as precaution. Because of this, I wouldn't do this here.